File: tests/fixtures/patterns-positive/prototype/mutB.ts
class Shape {
  color: string;
  x: number;
  y: number;
  constructor(color: string, x: number, y: number) {
    this.color = color;
    this.x = x;
    this.y = y;
  }
}

// POSITIVO (mutación B: sin clone(); copia de campos duplicada en 2 sitios).
function duplicateForLayer(shape: Shape): Shape {
  return new Shape(shape.color, shape.x, shape.y);
}
function duplicateForExport(shape: Shape): Shape {
  return new Shape(shape.color, shape.x, shape.y);
}

File: tests/fixtures/patterns-positive/prototype/mutB.py
class Shape:
    def __init__(self, color, x, y):
        self.color = color
        self.x = x
        self.y = y


# POSITIVO (mutación B: sin clone(); copia de campos duplicada en 2 sitios).
def duplicate_for_layer(shape):
    return Shape(shape.color, shape.x, shape.y)


def duplicate_for_export(shape):
    return Shape(shape.color, shape.x, shape.y)
